pkg/config/config.go

```diff
@@ -1143,7 +1143,7 @@ func (c *Config) FindHelperBinary(name string, searchPATH bool) (string, error)
 		return exec.LookPath(name)
 	}
 	configHint := "To resolve this error, set the helper_binaries_dir key in the `[engine]` section of containers.conf to the directory containing your helper binaries."
-	if len(c.Engine.HelperBinariesDir) == 0 {
+	if len(c.Engine.HelperBinariesDir.Values) == 0 {
```

Comment:
Is there a reason this uses Values vs Get()?

Reply:
`Get()` will never return `nil`, so it adds some cost.

Reply:
Why did you added this special handling to Get()? To me this seems to add unnecessary complexity. Returning nil shouldn't hurt or do podman/buildah tests fail becauase of it?

Reply:
buildah tests failed.  I can follow up on that if you want to.

Reply:
Although it may be safer to let `Get()` return a real copy in the future.

Reply:
I am fine with it but as you said it does extra allocations which means worse performance and memory footprint. It won't be noticeable (I assume) so it is not worth to block/argue about it but many small things start adding up at some point. Copying would definitely make it more expensive with the advantage that the API will be much safer with it.
The main issue is that technically callers can bypass that and just use `Values` directly so it is impossible to provide a safe API. Ideally go would allow us to keep the fields private but that won't work with the toml Unmarshaller obviously.

Reply:
I had the very same line of thoughts.  To really avoid callers abusing containers.conf, the loading must be made private and then we can add a carefully curated API (as we do in registries.conf).